rest/taskana-rest-spring/src/test/java/pro/taskana/workbasket/rest/WorkbasketControllerIntTest.java

```diff
@@ -311,4 +312,22 @@ void should_ThrowException_When_ProvidingInvalidFilterParams() {
         .extracting(ex -> ((HttpClientErrorException) ex).getStatusCode())
         .isEqualTo(HttpStatus.BAD_REQUEST);
   }
+
+  @Test
+  void test() {
+    String url = restHelper.toUrl(RestEndpoints.URL_WORKBASKET) + "?type=GROU";
+    HttpEntity<String> auth = new HttpEntity<>(restHelper.getHeadersAdmin());
+
+    ThrowingCallable httpCall =
+        () -> {
+          TEMPLATE.exchange(url, HttpMethod.GET, auth, WORKBASKET_SUMMARY_PAGE_MODEL_TYPE);
+        };
+
+    assertThatThrownBy(httpCall)
+        .isInstanceOf(BadRequest.class)
+        .hasMessageContaining(
+            "\"status\":400,\"error\":\"BAD_REQUEST\",\"exception\":\"org.springframework.web."
+                + "method.annotation.ModelAttributeMethodProcessor$1\",\"message\":\"org."
+                + "springframework");
```

Comment:
Better: Please verify that part of the message which says that "GROU" is invalid

Reply:
That's what I was attempting to do. Unfortunately I realized that only the first part of the message is displayed / checked without that information.